.gitignore

```diff
@@ -1,3 +1,4 @@
+.*.sw[op]
```

Comment:
It is. In this case, it is ignoring any gvim backup file (those starting with a dot, and ending with .swp or .swo).